common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -1,10 +1,7 @@
 #!/bin/bash
```

Comment:
Since all the bashisms in this file have been removed, we could change this to `/bin/sh`.

However, I'm in favour of standardising to use `/bin/bash` even in POSIX-compliant shell scripts. It's all too easy for a bashism to sneak in, and it doesn't become apparent until someone runs that specific code path in a shell that doesn't support that particular bashism.

Only if we would add scripts that are really intended to be used on released products, they should use POSIX shell. However, I don't think it should be necessary to have any prplMesh scripts in released products.